SimPEG/objective_function.py

```diff
@@ -301,24 +389,32 @@ def __getitem__(self, key):
 
     @property
     def multipliers(self):
-        """
-        Multipliers for each objective function
+        r"""Multipliers for the objective functions.
+
+        For a composite objective function :math`\phi` that is a weighted sum of
+        objective functions :math:`\phi_i` with multipliers :math:`c_i` such that
+
+        .. math::
+            \phi = \sum_{i = 1}^N c_i \phi_i
+
+        this method returns the multipliers :math:`c_i` in order.
```

Comment:
I think this line could be misleading users to think that multipliers are returned in ascending/descending order. I'm adding this suggestion, let me know what do you think.
```suggestion
        This method returns the multipliers :math:`c_i` in the same order of the ``objfcts``.
```